src/game/client/draw/drawable.cpp

```diff
@@ -638,13 +655,13 @@ void Drawable::Draw_Bombed(IRegion2D const *region)
     static const NameKeyType key_StickyBombUpdate = g_theNameKeyGenerator->Name_To_Key("StickyBombUpdate");
     StickyBombUpdate *update = (StickyBombUpdate *)object->Find_Update_Module(key_StickyBombUpdate);
 
-    if (update && update->Get_Target_Object()) {
+    if (update != nullptr && update->Get_Target_Object() != nullptr) {
         if (update->Has_Die_Frame()) {
             if (Get_Icon_Info()->anims[ICON_BOMB_TIMED] == nullptr) {
-                Get_Icon_Info()->anims[ICON_BOMB_TIMED] =
```

Comment:
Please put assert here.
```
captainslog_dbgassert(Get_Icon_Info()->anims[ICON_BOMB_REMOTE] == nullptr, "Must be null, otherwise would leak here");
```

Reply:
fixed.